### ISSUE TITLE
Ratio of FIB to Q&A items is much higher than in "real" PBE testing

### ISSUE THREAD

**Issue by ProfessorO** [Deadpikle/pathfinder-bible-experience-prep-quiz-master#33]
I believe the fraction of items that are FIB questions in the regional events has been around 10%. The current ratio generated by this engine appears to be roughly 50%.

**Comment by Deadpikle** [comment]
The default ratio is 30%. The default can be changed to 10%, but it's user-configurable. Are you finding somewhere where this % is not used correctly?

**Comment by Deadpikle** [comment]
https://github.com/Deadpikle/pathfinder-bible-experience-prep-quiz-master/blob/master/views/user/quiz/quiz-setup.twig#L120

**Comment by ProfessorO** [comment]
Is that the spinbox labeled "% Blanks" on the Quiz Setup page? I was misinterpreting that as the percentage of words in a verse that are turned into blanks, while I'm referring to the fraction of all questions that are FIB rather than Q&A questions.

**Comment by Deadpikle** [comment]
Ohhhhh I see what you're saying. Yes, there are two different ideas here:

* The percentage of blanks in a FIB question (currently defaults to 30%, see my previous comment)
* The ratio of FIB to Q&A questions in a given quiz

The first case is handled, the second case is not. Are you saying that the ratio for a given quiz (where there are both Q&A and FIB) needs to be something like 90% Q&A, 10% FIB? Am I understanding that correctly?

Right now, the current code looks like this:

https://github.com/Deadpikle/pathfinder-bible-experience-prep-quiz-master/blob/208d87c6f4413a39ef6516bbe0901c3fd6cccc67/app/Services/QuizGenerator.php#L284-L329

In other words, FIB vs Q&A questions are chosen randomly. That's the code that would need updating for a different ratio of FIB to Q&A. (We should make this user configurable!)

**Comment by ProfessorO** [comment]
I'd totally missed your last response, Michael. Yes, you captured exactly what I was referring to. And I agree that it should be user-configurable.

**Comment by Deadpikle** [comment]
@ProfessorO Finally fixed and user-configurable!
